Find-and-Replace/Replace-line-break-with-paragraph-mark/.NET-Standard/Replace-line-break-with-paragraph-mark/Program.cs

```diff
@@ -0,0 +1,147 @@
+                        IterateParagraphItems(paragraph.Items);
+                        break;
+                    case EntityType.Table:
+                        //Iterate through table.
+                        IterateTable(bodyItemEntity as WTable);
+                        break;
+                    case EntityType.BlockContentControl:
+                        BlockContentControl blockContentControl = bodyItemEntity as BlockContentControl;
+                        //Iterate to the body items of Block Content Control.
+                        IterateTextBody(blockContentControl.TextBody);
+                        break;
+                }
+            }
+        }
+        /// <summary>
+        /// Iterate table child elements.
+        /// </summary>
+        private static void IterateTable(WTable table)
+        {
+            //Iterate the row collection in a table.
+            foreach (WTableRow row in table.Rows)
+            {
+                //Iterate the cell collection in a table row.
+                foreach (WTableCell cell in row.Cells)
+                {
+                    IterateTextBody(cell);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Iterate paragraph child elements.
+        /// </summary>
+        private static void IterateParagraphItems(ParagraphItemCollection paraItems)
+        {
+            for (int i = 0; i < paraItems.Count; i++)
+            {
+                Entity entity = paraItems[i];
+                //Check the element type by using EntityType.
+                switch (entity.EntityType)
+                {
+                    case EntityType.Break:
+                        Break breakItem = entity as Break;
+                        //Replace line break with paragraph mark.
+                        if (breakItem.BreakType == BreakType.LineBreak)
+                        {
+                            WParagraph ownerPara = breakItem.OwnerParagraph;
+                            int breakIndex = ownerPara.ChildEntities.IndexOf(breakItem);
+                            int paraIndex = ownerPara.OwnerTextBody.ChildEntities.IndexOf(ownerPara);
+
+                            //Create new paragraph by cloning the existing paragraph.
+                            WParagraph newPara = ownerPara.Clone() as WParagraph;                           
+                            //Remove paragraph child entities from the index of break item.
+                            while (breakIndex <ownerPara.ChildEntities.Count)
+                            {
+                                ownerPara.ChildEntities.RemoveAt(breakIndex);                                
+                            }            
+                            int newParaItemsCount = ownerPara.ChildEntities.Count;
+                            // Remove paragraph child entities from index zero upto the end of paragraph break item.
+                            while (newParaItemsCount + 1 != 0)
```

Comment:
As you aware about the index of break in cloned paragraph, modify logic to remove from 0th index to index of break in cloned paragraph.
Modify while loop to remove items until specific index.